src/api/models/problem.model.ts

```diff
@@ -41,6 +42,10 @@ const problemSchema = new Schema<ProblemInterface>(
       type: Boolean,
       required: true
     },
+    fileName: {
+      type: String,
+      required: false
```

Comment:
Shouldn't this be true as this is required?

Reply:
I left it as false since previously created problems don't have a set file name. I wanted to prevent any random instances that updating the schema for a problem without a file name from failing.

Reply:
I see. Agree.